example_scoring.py

```diff
@@ -0,0 +1,17 @@
+from mw.api import Session
+
+from revscoring.extractors import APIExtractor
+from revscoring.languages import english
+from revscoring.scorers import MLScorerModel
+
+api_session = Session("https://en.wikipedia.org/w/api.php")
+extractor = APIExtractor(api_session, english)
+
+filename = "models/reverts.halfak_mix.trained.model"
```

Comment:
Maybe add a comment mentioning that this file would have been generated by some other command (linking to an example?)